### ISSUE TITLE
Loss value is "nan" during training using custom dataset

### ISSUE THREAD

**Issue by deeplearner93** [keras-team/autokeras#1474]
Loss value is "nan" during training using custom dataset

```
Trial 1 Complete [00h 05m 28s]
val_loss: nan

Best val_loss So Far: nan
Total elapsed time: 00h 05m 28s
INFO:tensorflow:Oracle triggered exit
Epoch 1/2
4/4 [==============================] - 160s 36s/step - loss: 315890295450641088.0000 - mean_squared_error: 315890295450641088.0000
Epoch 2/2
4/4 [==============================] - 143s 32s/step - loss: nan - mean_squared_error: nan
INFO:tensorflow:Assets written to: .\image_regressor\best_model\assets
```

**Comment by haifeng-jin** [comment]
Do you get nan for all the trials?
How many trials did you set?
If all the trials are nan, would you please share your dataset with us. We may further inspect.

**Comment by animecomico** [comment]
Hi, I have the same problem.
During Training the loss is nan after some trials
